CAInstructions.markdown

```diff
@@ -93,7 +93,7 @@ Add the root certificate and key to the destined folders:
 Create a \<user\> key and a certificate signing request in one step: (Enter \<usr_common_name\> when prompted and use a random passphrase)
 
 ```
-    openssl req -newkey rsa:4096 -keyout <usr_common_name>.key -out <usr_common_name>.csr -config openssl.cnf -days 3650
+    openssl req -newkey rsa:4096 -keyout <usr_common_name>.key -out <usr_common_name>.csr -config <openssl_configuration_file> -days 3650
```

Comment:
May be use PSC/openssl.cnf ?

As we have used openssl.cnf in Configuration file section:

'''
Create a CA configuration file (refer PSC/openssl.cnf). This is optional, but is very convenient while issuing multiple certificates. 
 
 Edit openssl.cnf, and change the following fields: (Here, directory \<usr\> must be directory TS, CP, or DP depending on the keys you want to generate.)
'''